test/gtest/ucs/test_event_set.cc

```diff
@@ -70,6 +75,8 @@ UCS_TEST_F(test_event_set, ucs_event_set_read_thread) {
     int pipefd[2];
     ucs_sys_event_set_t *event_set = NULL;
     ucs_status_t status;
+    ucs_event_set_extra_arg args[] = { (void*)UCS_EVENT_SET_EXTRA_STRING,
+                                       (void*)&UCS_EVENT_SET_EXTRA_NUM };
```

Comment:
What do you think about this declaration?

Reply:
if I recall it correctly, such syntax isn't allowed in C++
but I agree to have it if C++ compiler will be happy with that :)